cmd/gitserver/server/server_test.go

```diff
@@ -667,6 +666,13 @@ func TestCloneRepo(t *testing.T) {
 	if wantCommit != gotCommit {
 		t.Fatal("failed to clone:", gotCommit)
 	}
+	gitserverRepo, err := db.GitserverRepos().GetByName(ctx, repoName)
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
nit:
Testify's witness here: _"Do you have a moment to talk about our Lord and LoC Saver `require`?"_
```suggestion
	require.NoError(t, err)
```